dask/array/tests/test_linalg.py

```diff
@@ -910,3 +910,13 @@ def test_norm_implemented_errors(shape, chunks, axis, norm, keepdims):
     if len(shape) > 2 and len(axis) == 2:
         with pytest.raises(NotImplementedError):
             da.linalg.norm(d, ord=norm, axis=axis, keepdims=keepdims)
+
+
+def test_biggish_svd():
```

Comment:
How long does this take? Mark with `slow`?

Reply:
Not long.  I've renamed the test.